### DISCUSSION TITLE
BRIDGE_BookClub 6th Book

### DISCUSSION THREAD

**Discussion by fkdl0048** [BRIDGE-DEV/BRIDGE_BookClub#125] [Show and tell]
# BRIDGE_BookClub 6th Book

6회차에 진행할 서적을 미리 추천 받아서 이후 토론이 끝나고 난 후 다수결로 결정하겠습니다.

공식적인 6회차 시작(OT)는 2024-05-23입니다.

6회차를 시작하기 전 결정해야 할 내용은 다음과 같습니다.

- 5회차 책인 오브젝트의 부록도 스프린트로 잡아서 더 진행하기
  - 약 80쪽 분량으로 2주의 싸이클이 더 필요합니다.
  - 다만, 지금까지 부록은 항상 개인적으로 진행하든 선택의 영역이었지만, 내용이 생각보다 깊이가 있어서 같이 읽는 것도 좋을 것 같습니다.
- 논의 때 말씀드린 객체지향 설계에 관한 마무리로 같이 페어프로그래밍(도메인에 대한 설계 논의)를 진행할 것인지
  - 이 부분도 추가로 2주의 싸이클이 더 소모되거나, 마지막 날에 더 진행하게 되면 시간이 많이 소모됨
- 6회차 책에 대한 추천

현재 5회차까지 진행된 책은 다음과 같습니다.

- [1회차 클린코드](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3ACleanCode+)
- [2회차 리팩터링](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3ARefactoring+)
- [3회차 클린코더](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3A%22The+Clean+Coder%22+)
- [4회차 객체지향의 사실과 오해](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3A%22%EA%B0%9D%EC%B2%B4%EC%A7%80%ED%96%A5%EC%9D%98+%EC%82%AC%EC%8B%A4%EA%B3%BC+%EC%98%A4%ED%95%B4%22+)
- [5회차 오브젝트](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3A%EC%98%A4%EB%B8%8C%EC%A0%9D%ED%8A%B8)

항상 책을 해당 회차 진행원들과 회의를 통해 결정하기 때문에 이번 5회차에 참여하신 분들은 평소에 읽고 싶었거나 개인적 성장에 도움이 되는 책을 아래 코멘트로 적어주시면 됩니다.

*추천은 참여인원이 아니더라도 가능합니다.*

책을 선정하는 기준은 따로 없고 그때그때 달라지니 편하게 말씀해 주셔도 됩니다.

*개발 관련 or 게임 개발 서적으로 한정합니다.*

## 정리

위 3가지에 대해서 개인적인 생각을 적어주시면 됩니다.

**Comment by wooyn730** [comment]
부록도 스프린트로 잡는 거 좋습니다!

페어프로그래밍 해보고 싶은 마음은 있지만 제가 잘 못할 거 같아서 폐가 될까봐 망설여지네요..🥲

책은 요렇게 입니다
- 클린 코드
- 리팩터링
- 클린 아키텍처
- CODE 코드
- 실용주의 프로그래머
- 이펙티브 C#/C++
- 코드 작성 가이드
